### PR TITLE
Move small viewport behavior styles from core to Lumo

### LINKED ISSUE #133: Move small viewport behavior styles from core to Lumo
These styles should be removed or moved to the theme: https://github.com/vaadin/vaadin-context-menu/blob/master/src/vaadin-context-menu.html#L22-L38

Similar change was made to dropdown menu and date picker: https://github.com/vaadin/vaadin-date-picker/issues/508

This allows the theme to define the behavior. For example, the Material theme might choose not to display overlays in “fullscreen” on narrow viewports.

### PR DESCRIPTION
**Pull request by web-padawan**
Fixes #133

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-context-menu/134)

### CODE REVIEW

File: theme/lumo/vaadin-context-menu.html [hunk 1]
```diff
@@ -5,6 +5,24 @@
 <dom-module id="lumo-context-menu-overlay" theme-for="vaadin-context-menu-overlay">
   <template>
     <style include="lumo-menu-overlay">
+      [part="content"] {
+        background-color: #fff;
```
Comment (jouni) [inline theme/lumo/vaadin-context-menu.html]: This should be removed. `lumo-menu-overlay` specifies the necessary background colors.

Comment (web-padawan) [reply theme/lumo/vaadin-context-menu.html]: Done.

**Comment by tomivirkki** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-context-menu/134#-:-L4KvCOwUUQbIDZ1wCvK:bnfp4nl)*

**Comment by jouni** [comment]
Reviewed 2 of 2 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/vaadin/vaadin-context-menu/134)*

**Pull request merged**